src/modules/Ndvi/Module.js

```diff
@@ -46,7 +46,8 @@ module.exports = function Ndvi(options, UI) {
       format: input.format,
       image: options.image,
       inBrowser: options.inBrowser,
-      callback: modifiedCallback
+      callback: modifiedCallback,
```

Comment:
Huh, what is this change for?

Reply:
oh i see it's not a change, it's just a comma. Hm.

Reply:
yes.